cves/kernel/CVE-2020-10942.yml

```
@@ -169,8 +179,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: This vulnerability was actually originally found by Syzbot, a fuzzer that automatically tests the main Linux kernel branches. It was found so easily because the error boils down to a data field that lacks sanitation.
```

This is a really long line, can you indicate a multiline using "|"?

---

Noted and added! Thank you.
```suggestion
  note: |
   This vulnerability was actually originally found by Syzbot, a fuzzer that automatically tests the main Linux kernel 
   branches. It was found so easily because the error boils down to a data field that lacks sanitation.
```